src/result.cxx

```diff
@@ -502,6 +502,18 @@ pqxx::row::size_type pqxx::result::columns() const noexcept
 }
 
 
+int const pqxx::result::column_size(pqxx::row::size_type number) const
+{
+  return PQfsize(m_data.get(), number);
+}
+
+
+int const pqxx::result::column_mod(pqxx::row::size_type number) const
+{
+  return PQfmod(m_data.get(), number);
```

Comment:
What I said above about `PQfsize()` applies here as well, but with the added note that nothing tells me how we would ever put this modifier to use.  I'd rather leave this until someone shows up with an actual, concrete need that tells us how to deal with that.  If and when that happens, we may find that there's something much better that we can build on top of `PQfmod()`.  Until then...  _YAGNI._

Reply:
Well never mind what I said I guess.  :-)  And I see no way of detecting errors in this function, because all we know about the return value is that it's type-specific.  I guess this one can become `noexcept`.